src/migrations/Install.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+/**
+ * @link      https://github.com/lilt
+ * @copyright Copyright (c) 2022 Lilt Devs
+ */
+
+declare(strict_types=1);
+
+namespace lilthq\craftliltplugin\migrations;
+
+use craft\db\Migration;
+
+class Install extends Migration
+{
+    /**
+     * @inheritdoc
+     */
+    public function safeUp()
+    {
+        $this->createTable('{{%lilt_jobs}}', [
+            'id' => $this->primaryKey(),
+            'title' => $this->string()->null(),
+            'liltJobId' => $this->integer()->null(),
+            'status' => $this->string(50),
+            'elementIds' => $this->json(),
+            'sourceSiteId' => $this->integer()->unsigned()->notNull(),
+            'sourceSiteLanguage' => $this->string(50),
+            'targetSiteIds' => $this->json(),
+            'files' => $this->json(),
+            'dueDate' => $this->dateTime(),
+            'dateCreated' => $this->dateTime()->notNull(),
+            'dateUpdated' => $this->dateTime()->notNull(),
+            'uid' => $this->uid()
+        ]);
+
+        $this->createTable('{{%lilt_jobs_elements}}', [
+            'jobId' => $this->integer()->null(),
+            'elementId' => $this->integer()->null(),
+        ]);
+    }
```

Comment:
could you describe a bit about how these two tables will be used?

just some general questions

- what are all of the possible options for `lilt_jobs.status`?
- what is the difference between `lilt_jobs.elementIds` and `lilt_jobs.files`?
  - for both of these columns if says they are of the type JSON. could you share an example of what that JSON would look like? is it just a list of IDs?
- are the IDs from `lilt_jobs.elementIds` references to the `lilt_jobs_elements` table?

Reply:
>  what are all of the possible options for `lilt_jobs.status`?

[The whole list for is here ](https://github.com/lilt/craft-lilt-plugin/pull/1/files#diff-579371a9952c3ca6b1cae227d5e1ce4618d3f1b946cf4af418ae92d7c8bad3f5R125-R131)

All status are managed by code, not database. I am not sure if there will be a win from enum type. 

>  what is the difference between `lilt_jobs.elementIds` and `lilt_jobs.files`?

`lilt_jobs.files` contains all sent files to translate
`lilt_jobs.elementIds` contains ids of all selected elements to translate

>  for both of these columns if says they are of the type JSON. could you share an example of what that JSON would look like? is it just a list of IDs?

All of them are arrays which contains ids or files paths (we need it to save a job state). I would say we have json types because we don't have any other tables now (lilt_jobs_elements is a draft). But we will definitely will try to get rid of json type where it is possible. 

examples: 
```
["3"]
```
```
["/app/storage/runtime/temp/lilt-translate-file-1275821-2022-05-04.json"]
```

> are the IDs from `lilt_jobs.elementIds` references to the `lilt_jobs_elements` table?

The idea to remove elementIds (or keep as cache not sure yet) and use different table to manage all elements (because we need a status for each entry separated)
Actually lilt_jobs_elements should be a replacement for elementIds, I am working on it 

I will share a document with db structure with you